lib/helper/Playwright.js

```diff
@@ -1876,11 +1876,15 @@ class Playwright extends Helper {
    * {{> type }}
    */
   async type(keys, delay = null) {
+    // Always use page.keyboard.type for any string (including single character and national characters).
     if (!Array.isArray(keys)) {
       keys = keys.toString()
-      keys = keys.split('')
+      const typeDelay = typeof delay === 'number' ? delay : this.options.pressKeyDelay
+      await this.page.keyboard.type(keys, { delay: typeDelay })
+      return
     }
 
+    // For array input, treat each as a key press to keep modified keys working.
     for (const key of keys) {
```

Comment:
@mirao may you think if key here could be a letter with diacritical mark like `á` ?